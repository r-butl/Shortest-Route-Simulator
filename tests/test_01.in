8 12 80 80
0 7
0 0
1 1
2 0
3 0
4 1
5 0
6 0
7 1
0 1 41
0 2 49
0 3 52
0 7 102
1 5 65
1 4 38
2 4 27
3 6 61
4 6 37
5 7 23
4 7 57
6 7 54
