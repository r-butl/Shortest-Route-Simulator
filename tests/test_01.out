Verify Path: 1
133: 0 4 7 
